host/src/FunctionsNetHost.Shared/EnvironmentVariables.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace FunctionsNetHost.Shared
+{
+    public static class EnvironmentVariables
+    {
+        /// <summary>
+        /// The environment variable which is used to specify the specialized (function app payload) entry assembly.
+        /// </summary>
+        public const string SpecializedEntryAssembly = "AZURE_FUNCTIONS_FUNCTIONSNETHOST_SPECIALIZED_ENTRY_ASSEMBLY";
```

Comment:
Does "AZURE_FUNCTIONS_NETHOST_SPECIALIZED_ENTRY_ASSEMBLY" make more sense? Having Functions in there twice feels redundant

Reply:
Yes, that is even better. Switched to "AZURE_FUNCTIONS_NETHOST_SPECIALIZED_ENTRY_ASSEMBLY"